modules/comparison-of-solutions-to-provision-node-local-storage.adoc

```diff
@@ -0,0 +1,172 @@
+
+|Support for object storage ^[1]^ |No |No |No
+
+|Available filesystems |`ext4`, `xfs` |`ext4`, `xfs` |Any mounted system available on the node is supported.
+
+|====
+[.small]
+--
+1. None of the solutions ({lvms}, LSO, and HPP) provide support for object storage. Therefore, an S3 object storage solution, such as `MultiClusterGateway` from the Red Hat OpenShift Data Foundation, is needed. All of the solutions can serve as underlying storage providers for the S3 object storage solutions.
+--
+
+[id="comparing-core-functionalities"]
+== Comparison of the support for core functionalities
+
+The following table compares how {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) support core functionalities for provisioning local storage:
+
+.Comparison of the support for core functionalities
+[cols="6a,5a,5a,5a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Support for automatic file system formatting |Yes |Yes |N/A
+
+|Support for dynamic provisioning |Yes |No |No
+
+|Support for using software Redundant Array of Independent Disks (RAID) arrays 
+|Yes 
+
+Supported on 4.15 and higher versions.
+
+|Yes |Yes
+
+|Support for transparent disk encryption |Yes 
+
+Supported with 4.16 and higher versions.
+
+|Yes |Yes
+
+|Support for volume based disk encryption |No |No |No
+
+|Support for disconnected installation |Yes |Yes |Yes
+
+|Support for PVC expansion |Yes |No |No
+
+|Support for volume snapshots and volume clones |Yes |No |No
+
+|Support for thin provisioning |Yes
+
+Devices are thin-provisioned by default.
+
+|Yes
+
+You can configure the devices to point to the thin-provisioned volumes
+
+|Yes
+
+You can configure a path to point to the thin-provisioned volumes.
+
+|Support for automatic disk discovery and setup |Yes
+
+Automatic disk discovery is available during installation and runtime. You can also dynamically add the disks to the `LVMCluster` CR to increase the storage capacity of the existing storage classes.
+
+|Technology Preview
+
+Automatic disk discovery is available during installation. 
+
+|No
+
+|====
+
+[id="comparing-performance-and-isolation-boundary"]
+== Comparison of performance and isolation capabilities
+
+The following table compares the performance and isolation capabilities of {lvms}, Local Storage Operator (LSO), and HostPath Provisioner (HPP) in provisioning local storage.
+
+.Comparison of performance and isolation capabilities
+[cols="5a,6a,6a,6a",options="header"]
+|====
+
+|Functionality|LVM Storage|LSO |HPP
+
+|Performance
+
+|I/O speed is shared for all workloads that use the same storage class. 
+
+Block storage allows direct I/O operations. 
+
+Thin provisioning can affect the performance. 
+
+|I/O speed is shared for all workloads that use the same storage class. 
+
+Block storage allows direct I/O operations.
+
+|I/O speed is shared for all workloads that use the same storage class. 
+
+The restrictions imposed by the underlying filesystem can affect the performance.
```

Comment:
>  can affect the performance
Will the typical user know _what_ performance can be affected?

Reply:
I think in this context we are referring to "I/O speed". So it means ".. underlying filesystem can affect the I/O speed."

But I'll let @jakobmoellerdev comment on this!

Reply:
It looks good to me :)